src/chat.rs

```diff
@@ -1501,7 +1510,7 @@ pub unsafe fn remove_contact_from_chat(
             } else {
                 /* we should respect this - whatever we send to the group, it gets discarded anyway! */
```

Comment:
this comment belongs to the `if !(is_contact_in_chat())` failure above.

Reply:
maybe we can just remove the comment, i think it is kind of obvious that ppl that are not or not longer member of a group cannot do changes.